Tests/Units/BuilderTest.php

```diff
@@ -39,11 +39,15 @@ public function testThumbs()
     {
         $baseUrl = '//img.mapado.net/2014/1/1/my-image';
         $url = $this->manager->buildUrl($baseUrl, 200);
-        $this->assertSame($baseUrl . '_thumbs/200', $url);
+        $this->assertSame($baseUrl . '_thumbs/200-0', $url);
+
+        $baseUrl = '//img.mapado.net/2014/1/1/my-image';
+        $url = $this->manager->buildUrl($baseUrl, 0, 200);
+        $this->assertSame($baseUrl . '_thumbs/0-200', $url);
```

Comment:
Can you add a test with `null` and `null` for width and height ? and another one with `['allowwebp' => true]` ?